docs/source/getting_started/active_learning.rst

```diff
@@ -10,6 +10,13 @@ in your annotated dataset.
 
    Plot showing the different samples and how they perform on the clothing dataset.
 
+We designed Lightly Active Learning to give you the best performance while
+being very easy to use. Check out our tutorials to learn more:
+
+- :ref:`lightly-tutorial-active-learning-knn` 
```

Comment:
This is not -knn, but now -classification.

Reply:
This should be replaced with the detectron2 tutorial. Not sure why it's not here.
Please don't change the slugs or links of the tutorials. This will mess up any backlinks from other websites and break also internal links!

Reply:
Sorry, I did not know that, it happened in https://github.com/lightly-ai/lightly/commit/f58144f423a3fc3942f95aa4a18d892e27da01b4

Reply:
No worries. Will update